__tests__/app.test.js

```diff
@@ -479,3 +473,73 @@ describe("/api/articles/:article_id/comments", () => {
     });
   });
 });
+
+describe("/api/comments/:comment_id", () => {
+  describe("DELETE", () => {
+    test("204: successfully deletes the comment and returns no content", () => {
+      return request(app)
+        .delete("/api/comments/1")
+        .expect(204)
+        .then(({ body }) => {
+          return db.query("SELECT * FROM comments WHERE comment_id = 1");
+        })
+        .then(({ rows }) => {
+          expect(rows.length).toBe(0);
+        });
+    });
+
+    test("404: responds with an error when trying to delete a non-existent comment", () => {
+      return request(app)
+        .delete("/api/comments/9999")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("404 - Not Found: Comment not found");
+        });
+    });
+
+    test("400: responds with an error when comment_id is invalid", () => {
+      return request(app)
+        .delete("/api/comments/invalid_id")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("400 - Bad Request: invalid_id");
+        });
+    });
+
+    test("404: responds with an error when comment_id is missing", () => {
+      return request(app)
+        .delete("/api/comments/")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("404 - Not Found: Endpoint does not exist");
+        });
+    });
+
+    test("400: responds with an error when comment_id is a negative number", () => {
+      return request(app)
+        .delete("/api/comments/-1")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("400 - Bad Request: invalid_id");
+        });
+    });
+
+    test("400: responds with an error when comment_id is a decimal number", () => {
+      return request(app)
+        .delete("/api/comments/1.5")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("400 - Bad Request: invalid_id");
+        });
+    });
+
+    test("400: responds with an error when comment_id is zero", () => {
+      return request(app)
+        .delete("/api/comments/0")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("400 - Bad Request: invalid_id");
+        });
```

Comment:
We do not need to check if the id is negative or a decimal :)

Reply:
Thank you very much! Super helpful!